mxnet-android/README.md

```diff
@@ -0,0 +1,197 @@
+# DJL MXNet Android Template
+
+### Introduction
+This template includes 1. how you build the mxnet native library(*libmxnet.so*) for android
+2. how you use MXNet native library(*libmxnet.so*) in Android Studio with DJL to run a simple image classification.
+DJL team also prepare an unofficial prebuilt `libmxnet.so` on 4 common Android ABI.
```

Comment:
Does MXNet not release android compatible binary ?

Reply:
no